filaments/elegoo.json

```diff
@@ -55,6 +55,32 @@
                     "hex": "b5b7b8"
                 }
             ]
+        },
+		{
+            "name": "ASA {color_name}",
```

Comment:
Remove "ASA" from this field

Reply:
Better? My assumption was that it would conflict with the identical color_name section above for PLA. If that is incorrect, my mistake.

Reply:
Yup, the color_names are only specific to that material, so it won't be a conflict